packages/esm-patient-list-app/src/patient-list-list/patient-list-table.component.tsx

```diff
@@ -90,6 +86,51 @@ const PatientListTableContainer: React.FC<PatientListTableContainerProps> = ({
     goTo(1);
   };
 
+  // Handles marking patient starred
+  const [starredLists, setStarredLists] = useState([]);
+  const [starhandleTimeout, setStarHandleTimeout] = useState(null);
+  const { user: currentUser, mutateUser } = useCurrentUser();
+  const setInitialStarredLists = useCallback(() => {
+    const starredPatientLists = currentUser?.userProperties?.starredPatientLists ?? '';
+    setStarredLists(starredPatientLists.split(','));
+  }, [currentUser?.userProperties?.starredPatientLists, setStarredLists]);
+  const updateUserProperties = () => {
+    const starredPatientLists = starredLists.join(',');
+    const userProperties = { ...(currentUser?.userProperties ?? {}), starredPatientLists };
+    starPatientList(currentUser?.uuid, userProperties)
+      .then(() => mutateUser())
+      .catch(() => {
+        setInitialStarredLists();
+        showToast({
+          description: 'Marking patient lists starred/ unstarred failed',
```

Comment:
It would be great if this was translatable.

```suggestion
          description: 'Marking patient lists starred / unstarred failed',
```